modules/ITK/cli/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+if(MSVC11) #i.e. Visual Studio 2012
+	# Fix for VS2012 that has _VARIADIC_MAX set to 10. Don't set too high because it increases compiler memory usage / compile-time.
+	add_definitions( -D_VARIADIC_MAX=10 )
```

Comment:
the content of this 'if statement' should be moved in the main CMakeLists.txt else you would copy paste this code for every module.
@marcelluethi btw, I thought this code was already somewhere in statismo?